xwiki-platform-core/xwiki-platform-messagestream/xwiki-platform-messagestream-ui/src/main/resources/Main/MessageSenderMacro.xml

```diff
@@ -943,7 +943,7 @@ Valid values are: 'everyone', 'followers', 'group' or 'user'. </description>
     <property>
       <description>Comma separated list of visibility options that the macro should allow the user to choose from.
 
-This list should be a sublist of the defualt ones: 'everyone', 'followers', 'group', 'user'.</description>
+This list should be a sublist of the default ones: 'everyone', 'followers', 'group', 'user'.</description>
```

Comment:
No translation key for that?

Reply:
Just corrected a typo when I saw one :) The key `rendering.macro.messageSender.parameter.visibilityOptions.description`is defining the same sentences without the line break. I don't know if it's a good idea to use it here.

Reply:
Ok, might be something to check but out of the scope of your PR anyway.